packages/components/src/VirtualizedList/CellDatetime/CellDatetime.component.js

```diff
@@ -3,15 +3,36 @@ import React from 'react';
 import classnames from 'classnames';
 import { distanceInWordsToNow, format } from 'date-fns';
 import invariant from 'invariant';
+import enLocale from 'date-fns/locale/en';
+import frLocale from 'date-fns/locale/fr';
+import jaLocale from 'date-fns/locale/ja';
 
 import styles from './CellDatetime.scss';
+import { getCurrentLanguage } from '../../translate';
 
 const DATE_TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
+export function getDateFnsLocale(language) {
```

Comment:
What do you think of having this function and the locales imports in `../../translate` instead, in case we will use it in other components? If yes, we you could do `getCurrentDateFnsLocale`